packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -199,7 +208,9 @@ export class FabricTestLedgerV1 implements ITestLedger {
     const fnTag = `${this.className}#getConnectionProfileOrg1()`;
     const cInfo = await this.getContainerInfo();
     const container = this.getContainer();
-    const ccpJsonPath = "/fabric-samples/first-network/connection-org1.json";
+    const ccpJsonPath = this.getFabricVersion().valueOf().includes("1.4.")
```

Comment:
Use [this](https://www.npmjs.com/package/compare-versions#human-readable-compare) (or something like it) to cover all possible integers for the supported versions.
It is okay for us to only support Fabric 1.4.x, we can throw for 1.3, 1.2, ... 0.1, etc...
Throw exceptions if the provided fabric version is not in the supported range.
Cover cases of major version being above 2.x (when Fabric 3.x inevitably hits, we want at least our code to fail gracefully instead of having undefined side effects/silent bugs). So if the version is greater or equal (GTE) than 3.0.0 then you can either print a warning saying that we are falling back to using the 2.x paths and this may not work (warning log level) OR you can just throw saying 3.x is not supported (we don't know yet which one is more appropriate because we have no idea if they'll change the paths again or not).

Reply:
Incorporated the changes. Please have a re-look.